nntrainer/models/neuralnet.cpp

```diff
@@ -644,30 +700,27 @@ void NeuralNetwork::load(const std::string &file_path,
   auto v = split(file_path, reg_);
 
   if (exec_mode == ExecutionMode::INFERENCE && swap_mode) {
-
+    std::cout << "At NeuralNetwork load" << std::endl;
     model_graph.setFsuWeightPath((v.size() == 2) ? v[1] : v[0]);
 
     std::vector<std::pair<size_t, size_t>> file_offset;
-    // size_t start_from = sizeof(unsigned short);
     size_t start_from = 0;
-    for (auto node : model_graph.getLayerNodes()) {
-      auto weights = node->getRunContext().getWeights();
+
+    for (auto iter = model_graph.cbegin(); iter != model_graph.cend(); iter++) {
+      auto weights = (*iter)->getRunContext().getWeights();
       for (auto weight : weights) {
-        auto dim = weight->getDim();
-        size_t size =
-          dim.getDataTypeSize() * dim.getDataLen(); // + scale_size * float
+        auto dim = weight->getVariable();
 
-        // auto var_t = weight->getVariable();
-        // size_t size = var_t.getMemoryBytes();
+        size_t size =
+          dim.getMemoryBytes(); // dim.getDataTypeSize() * dim.getDataLen();
 
-        // there is uint16 to save quantization bit for each tensor
         file_offset.emplace_back(std::make_pair(start_from, size));
-        // start_from += size + sizeof(unsigned short);
         start_from += size;
       }
     }
     model_graph.setWeightOffset(file_offset);
   }
+
```

Comment:
make sure the model seWeightOffset function called once. It will remove original bin file if it called twice.